conda_smithy/configure_feedstock.py

```diff
@@ -1204,6 +1217,8 @@ def _load_forge_config(forge_dir, exclusive_config_file):
     for platform in ["linux_aarch64", "linux_ppc64le"]:
         if config["provider"][platform] == "default":
             config["provider"][platform] = "azure"
+    if config["provider"]["linux_ppc64le"] == "native":
+        config["provider"][platform] = "travis"
```

Comment:
Can we document these aliases somewhere?

Reply:
They should be documented here: https://github.com/conda-forge/conda-forge.github.io/blob/master/src/conda-forge-yml.rst  But that is outside of this particular PR